docs/system/system-metrics.md

```diff
@@ -143,6 +144,34 @@ This metric is [recommended][MetricRecommended].
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+### Metric: `system.cpu.utilization`
+
+This metric is [recommended][MetricRecommended].
+
+<!-- semconv metric.system.cpu.utilization -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability | Entity Associations |
+| -------- | --------------- | ----------- | -------------- | --------- | ------ |
+| `system.cpu.utilization` | Gauge | `1` | Fraction of total CPU time spent in active modes, aggregated across all system CPU cores. [1] | ![Development](https://img.shields.io/badge/-development-blue) | `host` |
+
+**[1]:** This metric provides a standardized and opinionated definition of CPU utilization.
+It is defined as the proportion of total system CPU time spent in non-idle and non-iowait modes — that is, the time spent in user, system, nice, interrupt, and steal modes.
+It is computed by aggregating `cpu.time` across all logical CPUs over a measurement interval,
+and dividing the result by the elapsed time and the `system.cpu.logical.count` to produce a normalized,
+system-wide utilization value between 0.0 and 1.0: `sum(rate(cpu.time{mode != "idle" and mode != "iowait"})) / system.cpu.logical.count`.
+This represents time spent actively executing work, including time delayed by the hypervisor (steal), which reflects CPU demand in virtualized environments.
```

Comment:
This calculation will be challenged by end users: "why don't you take into account iowait?", "why don't you remove steal time?", etc. People will never come to an agreement, because different they have different use cases and needs.